src/Inviqa/StockIndicatorExport/Domain/Model/StockIndicator.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Inviqa\StockIndicatorExport\Domain\Model;
+
+use InvalidArgumentException;
+
+final class StockIndicator
+{
+    private const LOW_STOCK_THRESHOLD = 10;
+
+    /** @var string */
+    private $value;
+
+    public static function fromString(string $value): StockIndicator
+    {
+        if (!in_array($value, ['RED', 'YELLOW', 'GREEN'])) {
```

Comment:
Shall we have these level colour indicators as constants?

Reply:
yep make sense :)